Resources/public/js/views/services/plugins/ez-searchplugin.js

```diff
@@ -326,32 +326,48 @@ YUI.add('ez-searchplugin', function (Y) {
          *
          * @method _loadContentTypeForLocations
          * @protected
-         * @param {Array|Null} locationStructArr
+         * @param {Array|Null} structArray
+         * @param {Function} getContentTypeIdFunc
          * @param {Function} callback
          */
         _loadContentTypeForStruct: function (structArray, getContentTypeIdFunc, callback) {
             var tasks = new Y.Parallel(),
-                loadContentTypeError = false;
+                loadContentTypeError = false,
+                contentTypList = [];
+
+            // Get list of content types ids
+            var contentTypeIds = Y.Array.map(structArray, function (struct, i) {
```

Comment:
CS: the variable should declared in the `var` block above (one `var` block per function)